lib/experimental/Widgets/Content/CalendarEvent/index.tsx

```diff
@@ -0,0 +1,149 @@
+import { Icon, IconType } from "@/components/Utilities/Icon"
+import { DateAvatar } from "@/experimental/Information/Avatars/DateAvatar"
+import { RawTag } from "@/experimental/Information/Tags/RawTag"
+import { Tooltip } from "@/experimental/Overlays/Tooltip"
+import { ChevronRight } from "@/icons/app"
+import { forwardRef } from "react"
+
+type Tag = {
+  icon: IconType
+  label: string
+  description: string
+}
+
+export interface CalendarEventProps {
+  label?: string
+  title: string
+  subtitle: string
+  description: string
+  color: string
+  isPending: boolean
+  tags?: Tag[]
+  action?: {
+    title: string
+    icon: IconType
+    onClick: () => void
+  }
+  fromDate?: Date
+  toDate: Date
+  showBackground?: boolean
```

Comment:
maybe `noBackground` and have it `false` by default?

It is nicer when using the component. Instead of:

```
<CalendarEvent showBackground={false} />
```

I can do
```
<CalendarEvent noBackground />
```